ddtrace/debugging/_probe/remoteconfig.py

```diff
@@ -116,6 +116,7 @@ def probe(_id, _type, attribs):
             line=int(attribs["where"]["lines"][0]),
             name=attribs["metricName"],
             kind=attribs["kind"],
+            value=_compile_condition(attribs.get("value")),
```

Comment:
We should probably rename this to `_compile_expression` at this point

Reply:
renamed

Reply:
Just checking that we didn't miss a change here. the diff view is still showing the old name here.